rebar.config

```diff
@@ -243,7 +245,7 @@
             },
             {test_validator, [
               {relx, [
-                {release, {miner, git},
+                {release, {miner, {semver, "val"}},
```

Comment:
What is `"val"`? I don't see such miner branch or tag.

Reply:
fixed this https://github.com/helium/miner/commit/5a6b4ee56a9323af9a89b5766f6229de2bc5cd0b

Reply:
@andymck